docs/developer/architecture/logs.rst

```diff
@@ -197,6 +197,444 @@ overview page.
+chart values:
+
+For Fluent Bit, in ``charts/loki-stack.yaml``:
+
+.. code-block:: yaml
+
+  fluent-bit:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+For Loki, in ``charts/loki-stack.yaml``:
+
+.. code-block:: yaml
+
+  loki:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+For Fluentd, in ``charts/fluentd.yaml``:
+
+.. code-block:: yaml
+
+  metrics:
+    serviceMonitor:
+      enabled: true
+      additionalLabels:
+        release: "prometheus-operator"
+
+We also need to define alert rules based on metrics exposed by these services.
+This is done deploying new ``PrometheusRules`` object in ``metalk8s-logging``
+Namespace.
+
+.. code-block:: yaml
+
+  apiVersion: monitoring.coreos.com/v1
+  kind: PrometheusRule
+  metadata:
+    labels:
+      release: prometheus-operator
+    name: loki.rules
+    namespace: metalk8s-logging
+  spec:
+    groups:
+    - name: loki.rules
+      rules: [TO BE DEFINED]
+
+.. note::
+
+  Didn't find any Prometheus Rules for Loki on the Internet.
+
+  TODO: Need to define them based on what's exposed.
+
+Grafana
+"""""""
+
+To be able to query logs from Loki, we need to add a Grafana datasource,
+this is done adding a ConfigMap ``loki-grafana-datasource`` in
+``metalk8s-monitoring`` Namespace as follows:
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: loki-grafana-datasource
+    namespace: metalk8s-monitoring
+    labels:
+      grafana_datasource: "1"
+  data:
+    loki-datasource.yaml: |-
+      apiVersion: 1
+      datasources:
+      - name: Loki
+        type: loki
+        access: proxy
+        url: http://loki.loki.svc.cluster.local:3100
+        version: 1
+
+To read the log we will also need a dashboard, adding a ConfigMap
+``loki-logs-dashboard`` in Namespace ``metalk8s-monitoring``:
+
+.. code-block:: yaml
+
+  apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: loki-logs-dashboard
+    namespace: metalk8s-monitoring
+    labels:
+      grafana_dashboard: "1"
+  data:
+    loki-logs.json: [TO BE DEFINED]
+
+.. note::
+
+  The ``grafana_datasource: "1"`` and ``grafana_dashboard: "1"`` labels are
+  what is used by the Prometheus Operator to retrieve datasource and dashboard
+  for Grafana.
+
+Iteration 1
+~~~~~~~~~~~
+
+The goal is to have a working log centralization system, with logs accessible
+from Grafana:
+
+* Deployment of the stack (without Fluentd)
+* Deploy Grafana datasource & dashboard
+* Document the log centralization system
+* Simple pre-merge test to ensure the log pipeline is working
+* Customization of Loki, Fluent Bit and Fluentd (editing the CSC ConfigMap)
+
+Iteration 2
+~~~~~~~~~~~
+
+* Deployment of Fluentd
+* Customization of Loki, Fluent Bit and Fluentd (simple + advanced modes)
+* Document how to configure these components
+* Define monitoring rules
```

Comment:
and display in MetalK8s Overview page

Reply:
Not sure if really needed in an Overview page, as it's not a critical component of the system. I'd rather have dedicated pages for the various "infra services" we deploy (auth, monitoring, logging).